src/Api/AdminConsole/Controllers/PoliciesController.cs

```diff
@@ -212,4 +215,16 @@ public async Task<PolicyResponseModel> Put(Guid orgId, PolicyType type, [FromBod
         var policy = await _savePolicyCommand.SaveAsync(policyUpdate);
         return new PolicyResponseModel(policy);
     }
+
+
+    [HttpPut("{type}/vnext")]
+    [RequireFeatureAttribute(FeatureFlagKeys.CreateDefaultLocation)]
+    [Authorize<ManagePoliciesRequirement>]
```

Comment:
I don’t think there’s much value in adding tests to this endpoint at this stage since we moved validations to use the attributes.

We can revisit this in future PRs for this same ticket.